### PR TITLE
Archive/more flexibility

### PR DESCRIPTION
**Pull request by bradvogel**

### CODE REVIEW

File: test/janitor.spec.js [hunk 1]
```diff
@@ -186,7 +194,7 @@ define([ 'html-janitor' ], function (HTMLJanitor) {
 
         var output = janitor.clean(el.outerHTML);
 
-        expect(output).toBe('<figure></figure>');
+        expect(output).toBe('');
```
Comment (bradvogel) [inline test/janitor.spec.js]: @rrees This is right - right?

File: src/html-janitor.js [hunk 2]
```diff
@@ -98,7 +98,13 @@
             isNotTopContainer;
 
       var nodeName = node.nodeName.toLowerCase();
-      var allowedAttrs = this.config.tags[nodeName];
+
+      var allowedAttrs;
```
Comment (rrees) [inline src/html-janitor.js]: Let's turn this into an assignment from a function.

File: test/janitor.spec.js (hunk 1 above)
Comment (rrees) [inline test/janitor.spec.js]: Yes, it looks right to me. I wonder why it was passing.

**Comment by rrees** [comment]
With small comment :+1:

File: test/janitor.spec.js (hunk 1 above)
Comment (bradvogel) [inline test/janitor.spec.js]: I'm not sure. It was a bug though.

**Comment by bradvogel** [comment]
@rrees Updated. Is that what you were thinking?

**Comment by rrees** [comment]
@bradvogel Yes, indeed. Looks very nice. I'll try and release as soon as I can.

**Comment by bradvogel** [comment]
Great, thanks!

**Pull request merged**